lib/sidekiq/processor.rb

```diff
@@ -119,18 +119,21 @@ def handle_fetch_exception(ex)
       nil
     end
 
+    def log_context(job_hash)
```

Comment:
I'm ambivalent on that point because that would mean teaching Logging about job_hashes rather than having a simple string-based interface. Maybe you have other ideas on implementation?